index.html

```diff
@@ -0,0 +1,223 @@
+                </ul></div>
+            <a class="button_card3" href="#">See Project</a>
+        </div>
+        <div class="card4">
+            
+            <h2>Multi-Post Stories</h2>
+            <div>
+              <ul class="client_card4">
+                <li><a class="client-name">CANOPY</a></li>
+                <li><div class="circle"></div></li>
+                  <li><a class="role">Back End Dev</a></li>
+                    <li><div class="circle"></div></li>
+                      <li><a class="year">2015</a></li>
+               </ul></div>
+            <p class="description_card4">A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div>
+                <ul class="categories_card4">
+                  <li><a href="#" class="ollinks">html</a></li>
+                    <li><a href="#" class="ollinks">css</a></li>
+                      <li><a href="#" class="ollinks">javascript</a></li>
+                </ul></div>
+            <a class="button_card4" href="#">See Project</a>
+        </div>
+
+ </section>
+    <section class="about_myself">
+        <h3>About <br> Myself</h3>
+        <p class="about_myself_paragraph"> Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me. </p>
+        <h2 class="about_myself_connect">LET'S CONNECT</h2>
+        <ul class="about_myself_social">
+          <li><a href="#"><img src="./images/github.svg" alt="github" class="github"></a></li>
+            <li><a href="#"><img src="./images/linkedinicon.svg" alt="linkedin" class="linkedin" ></a></li>
+            <li><a href="#"><img src="./images/two-finger.svg" alt="angelist" class="angelist" ></a></li>
+            <li><a href="#"><img src="./images/twitter.svg" alt="twitter" class="twitter"></a></li>
+            <li><a href="#"><img src="./images/medium.svg" alt="medium" class="medium" ></a></li>
+            
+        </ul>
+       
+        <button type="submit" class="get_resume" >
+         Get my resume</button>
+
+      <div class="list-items">
+        <ul>
+          <li><a href="#" class="languages">Languages</a></li></ul>
+            <div class="languages-vector">
+              
+            </div>
+            <div class="javascript">
+              <div class="imagejs"></div>
+              <h4 class="titlejs">JavaScript</h4>
+            
+            </div>
+
+            <div class="html">
+              <div class="imagehtml"></div>
+              <h4 class="titlehtml">HTML</h4>
+            
+            </div>
+              
+            <div class="css">
+              <div class="imagecss"></div>
+              
+              <div class="titlecss">CSS</div>
+            
+            </div>
+          
+         
```

Comment:
- [ **OPTIONAL** ] I think it would be better to remove all unnecessary empty lines to make your code clean and smart. I know you can do it :muscle: :muscle:.